### DISCUSSION TITLE
Running artillery from source

### DISCUSSION THREAD

**Discussion by vikramjb** [artilleryio/artillery#2535] [Artillery]
@hassy @bernardobridge @InesNi 

Is there any documentation on how I can run artillery from source? I'd like to try to running the latest main branch on some of the scripts I have.

**Comment by hassy** [comment]
This should work:

1. Clone the repo
2. Run `npm install` at the root of the repo
3. Run `packages/artillery/bin/run run myscript.yaml`

**Reply by hassy** [reply]
and if you just want to try the latest Artillery from `main` install the canary release with:

```
npm install -g artillery@canary
```

Every commit to `main` is published automatically under a `canary` tag.

**Comment by vikramjb** [comment]
Fantastic!, That works 👍 Thank you for that @bernardobridge @hassy
